src/components/Card.js

```diff
@@ -1,75 +1,94 @@
-// import { openModal } from "../utils/utils.js";
-
-// const previewImageModal = document.querySelector("#preview-image-modal");
-// const modalImage = previewImageModal.querySelector(".modal__preview-image");
-// const modalCaption = previewImageModal.querySelector(".modal__preview-title");
-
-export default class Card {
-  constructor({ cardData, handleImageClick }, cardSelector) {
-    this._name = cardData.name;
-    this._link = cardData.link;
+class Card {
+  constructor(
+    { name, link, likes, owner, _id },
+    cardSelector,
+    handleImageClick,
+    handleDeleteCardPopup,
+    handleLikeClick,
+    userID
+  ) {
     this._cardSelector = cardSelector;
     this._handleImageClick = handleImageClick;
+    this._imageName = name;
+    this._imageLink = link;
+    this._handleDeleteCardPopup = handleDeleteCardPopup;
+    this._cardOwnerID = owner._id;
+    this._cardId = _id;
+    this._userID = userID;
+    this._handleLikeClick = handleLikeClick;
+    this._likes = likes;
+  }
+
+  _setEventListeners() {
+    this._likeButton.addEventListener("click", () =>
+      this._handleLikeClick(this)
+    );
+    this._cardDeleteButton.addEventListener("click", () =>
+      this._handleDeleteCardPopup(this)
+    );
+
+    this._cardImage = this._cardElement.querySelector(".card__image");
+    this._cardImage.addEventListener("click", () =>
+      this._handleImageClick({ link: this._imageLink, name: this._imageName })
+    );
+    this._fillCardTemplate();
   }
 
   _getTemplate() {
     return document
       .querySelector(this._cardSelector)
-      .content.querySelector(".card__list-item")
+      .content.querySelector("#cards-template-list")
       .cloneNode(true);
   }
 
-  _fillCardData() {
-    this._cardImage.alt = this._name;
-    this._cardImage.src = this._link;
-    this._cardTitle.textContent = this._name;
-  }
-
-  _setEventListeners() {
-    this._likeButton.addEventListener("click", () => this._handleLikeIcon());
-    this._deleteButton.addEventListener("click", () =>
-      this._handleDeleteCard()
-    );
-
+  _fillCardTemplate() {
     this._cardImage = this._cardElement.querySelector(".card__image");
```

Comment:
`    this._cardImage = this._cardElement.querySelector(".card__image");
`

The image has been found in line 30 and it's accessible here. No need to search it again